include/z64kanfont.h

```diff
@@ -0,0 +1,43 @@
+#ifndef Z64KANFONT_H
```

Comment:
Should this file be `z64font.h` instead given everything here is `FONT_` and `Font` instead of Kanfont?

Reply:
fwiw I think system name is better cause its what people are interacting with more as they use the codebase. We already do this with for example environment vs kankyo (z64environment.h)

Reply:
After thinking about this I think you both guys are right, I'll rename it to z64font.h